code/modules/projectiles/updated_projectiles/ammunition.dm

```diff
@@ -100,7 +100,7 @@ They're all essentially identical when it comes to getting the job done.
 //This will attempt to place the ammo in the user's hand if possible.
 /obj/item/ammo_magazine/proc/create_handful(mob/user, transfer_amount)
 	var/R
-	if (current_rounds > 0)
+	if(current_rounds > 0)
```

Comment:
Mind doing an early return here for clarity? `if(current_rounds <= 0) return FALSE`